### DISCUSSION TITLE
remove of data cached by borg

### DISCUSSION THREAD

**Discussion by nbehrnd** [borgbackup/borg#7048] [Q&A] [answered]
Running borg backup with vorta as GUI in an instance of Linux Debian, I notice each time data are written into `~/.cache/borg` of the computer where borg and vorta are installed and started. For me, after removing temporary data (including system cache) with bleachbit and a completely recorded backup from vorta, `du -h` typically records close to 800MB in `~/.cache/borg`.  Because these data are in in a folder with name `borg`, I assume these data belong to borg.  So far, I did not observe difficulties to run vorta (and via vorta, borg) to record a new backup, restore a backup/a single file from a backup, compare two backups with each other, etc. *after* removing these intermediate data.

The vorta GUI offers to run some shell commands after a successful recording of a backup.  This possibly can be used to automatically remove directory `~/.cache/borg` accessible (including erasing) without elevated (sudo) privileges:

![remove_borg_cache](https://user-images.githubusercontent.com/19709084/191912532-1aeb6c4a-6707-4177-b440-a672fe2d7c65.png)

**My question:** does borg already include an option to remove the temporary data (which eventually could be invoked by vorta, too)?

Vorta's discussion board includes a thread where `~/.cache/borg` is mentioned ([link](https://github.com/borgbase/vorta/discussions/1340)), however the specific address to readthedocs.io just above the post's screen photo no longer is successfully resolved.  Commands described in the documentation [creation](https://borgbackup.readthedocs.io/en/stable/usage/create.html) and adjacent sections focus on generation and managing already recorded backups, *so far* I did not spot an entry relevant to my intent.

The observations refer to Linux Debian 12/bookworm with borg 1.2.2 and vorta 0.8.7, both directly installed from the OSes repositories.

**Comment by infectormp** [comment]
Did you try to find answer in the FAQ?
https://borgbackup.readthedocs.io/en/stable/faq.html#the-borg-cache-eats-way-too-much-disk-space-what-can-i-do
https://borgbackup.readthedocs.io/en/stable/internals/data-structures.html#cache

**Reply by nbehrnd** [reply]
The snippet of code the first address provides indeed was helpful, though it was not understood right away.  To ease how to trade the affairs in such a situation, especially for those used to use borg indirectly via vorta, I filed a PR (https://github.com/borgbackup/borg/pull/7049) so that the documentation is even more valuable than it already is.  Thank you.

**Comment by infectormp** [comment, marked as answer]
borg doesn't have option to remove the temporary data
